carla/models/catalog/catalog.py

```diff
@@ -41,8 +43,10 @@ def __init__(
             The directory in which to cache data; see :func:`get_models_home`.
         kws : keys and values, optional
             Additional keyword arguments are passed to passed through to the read model function
-        data : data.api.Data Class
-            Correct dataset for ML model
+        encode_normalize_data : bool, optional
+            If true, the model pipeline is used to build data.encoded, data.normalized and data.encoded_normalizd
+        use_pipeline : bool, optional
+            If true, the model uses a pipeline before predict and predict_proba to preprocess the input data.
         """
         self._backend = backend
 
```

Comment:
Can the model class keep the data? i.e. self._data = data

Reply:
Sounds like a good idea, @indyfree included this in the comment above, too.
Will add this feature in further commits 👍